.github/workflows/benchmark.yml

```diff
@@ -6,6 +6,8 @@ on:
     types: [ submitted ]
   schedule:
     - cron: '0 0 * * *'
+  # Manurally trigger
+  workflow_dispatch:
```

Comment:
I add the `workflow_dispatch` trigger so that we can manually bench specified branches.